include/logger/sink.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef SINK_HPP
+#define SINK_HPP
+
+#include "formatter.hpp"
+
+namespace logger {
+class ISink
+{
+public:
+  /**
+   * @brief Assigns the pointer to the formatter.
```

Comment:
Can you make this @brief better.. it should be pointer or any technical it should be explaination of the class telling whats its use

Reply:
Explain what this interface is for and what is sink

Reply:
Put an addition PR for this type correction in already mereged code in master